internal/resource/resource.go

```diff
@@ -259,10 +257,20 @@ func NewResource(ctx context.Context, slice *apiv1.ResourceSlice, index int) (*R
 		},
 	}
 
-	parsed, err := res.Parse()
+	hash := fnv.New64()
+	hash.Write([]byte(resource.Manifest))
+	res.ManifestHash = hash.Sum(nil)
+
+	parsed := &unstructured.Unstructured{}
+	err := parsed.UnmarshalJSON([]byte(resource.Manifest))
 	if err != nil {
 		return nil, fmt.Errorf("invalid json: %w", err)
 	}
+	if parsed.Object != nil {
+		delete(parsed.Object, "status")
```

Comment:
Reasoning behind this? A comment would be great

Reply:
I'm thinking we can just remove this code now that we have the resource version check? It protected against a case where the synthesizer produces resources that have non-nil status structs since they would never converge if the resource kind uses the status subresource.

Reply:
Well `go-synth-example` is failing so apparently I'm missing something lol

Reply:
I think the test was only failing because I didn't rebase on the commit that includes the resource version check. But regardless let's leave this as-is in this PR and maybe removed it later. Added a comment.